### DISCUSSION TITLE
Duplicate option on context menu

### DISCUSSION THREAD

**Discussion by barcar** [coderaiser/cloudcmd#402] [Ideas]
It would be really useful to have a Duplicate option on the context menu - alongside Copy/Paste. Currently there is no easy way to clone a file and rename the new copy (copy/paste to same folder complains about the path being the same).

Proposal would be that Duplicate prompts for a new path with the current name/path being offered as a starting point.

**Comment by coderaiser** [comment]
That’s a good idea, so you want file duplicated not to the current directory like in Mac OS Finder, but in any path? It can be solved easily with User Menu, take a look at [cookbook](https://github.com/coderaiser/cloudcmd/wiki/User-Menu-Cookbook).

**Reply by barcar** [reply]
Well, I actually want it duplicated in the same folder with a different name - like if I'm copying a template. I'll take a look at the user menu - thanks.

**Reply by coderaiser** [reply]
If you will have any questions on details of implementations - ask, and would be great if you share you implementation to cookbook, so other users can benefit :)
